.travis.yml

```diff
@@ -13,7 +13,7 @@ matrix:
       script:
         - make lint
         # Lint the first commit in the PR.
-        - git log $TRAVIS_COMMIT_RANGE --pretty=format:'%h' --no-merges | tail -1 | xargs npx core-validate-commit --no-validate-metadata
+        - echo 'Linting the commit message...' && git log $TRAVIS_COMMIT_RANGE --pretty=format:'%h' --no-merges | tail -1 | xargs npx -q core-validate-commit --no-validate-metadata
```

Comment:
Maybe. There is, however, a reliance on a Travis-specific environment variable....

Reply:
A script could be parameter based...
Or maybe move the logic into `core-validate-commit`.
It would be nice to be able to just do `npx core-validate-commit` and get an answer.